filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,54 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
+  license: CC-0
+doc-ref: https://github.com/AgentD/squashfs-tools-ng/blob/master/doc/format.txt
+seq:
+  - id: superblock
+    type: superblock
+types:
+  superblock:
+    seq:
+      - id: magic
+        contents: 'hsqs'
+      - id: inode_count
+        type: u4
+      - id: mod_time
+        type: u4
+      - id: block_size
+        type: u4
+      - id: frag_count
+        type: u4
+      - id: compressor
```

Comment:
Shluldn't it be a enum?

Reply:
Most likely. I am still learning the .ksy format. In the spec it is.
```
 |      |               | 1     | GZIP | just zlib deflate (no gzip headers!) |
 |      |               | 2     | LZO  |                                      |
 |      |               | 3     | LZMA | LZMA version 1                       |
 |      |               | 4     | XZ   | LZMA version 2 (no XZ headers!)      |
 |      |               | 5     | LZ4  |                                      |
 |      |               | 6     | ZSTD |                                      |
```

Reply:
btw, we have a lib of decompressors

Reply:
What for?

Reply:
zlib (raw + gzip), lzma (both versions), lz4, zstd

see https://github.com/kaitai-io/kaitai_compress